lab-james/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "lab-james",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js"
```

Comment:
Dont forget your other scripts for linter, and linter + mocha. Per lab instructions